modules/ding_seo/features/ding_seo_metatag_content/ding_seo_metatag_content.features.metatag.inc

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * @file
+ * ding_seo_metatag.features.metatag.inc
+ */
+
+/**
+ * Implements hook_metatag_export_default().
+ */
+function ding_seo_metatag_content_metatag_export_default() {
+  $config = array();
+
+  // Exported Metatag config instance: node.
+  $config['node'] = array(
```

Comment:
This is the default global config, so no need to export. I also believe we have the title-tag configured elsewhere so it will be overridden. 

Maybe it would be a good idea to move some of the config from panels to here, but I think we should concentrate on the schema related stuff here.

I will do manual merge and fix the export, so no need to change.

Reply:
Great if you'll do a manuelt merge.

Reply:
Is there a reason for having the features in a separate module? Otherwise I will move them into the main module ding_seo.

Reply:
No other reason other than it's the way we usually do it in Odense, keeping the features and code separate, so it's perfectly fine to move it into main module.

Reply:
I don't mind doing having features in a separate module if there's some good reason for it? The most important thing is that we're consistent :)

Reply:
It's fine if you move it into the main module.